src/docs/product/issues/issue-details/performance-issues/large-render-blocking-asset/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Large Render Blocking Asset"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/large-render-blocking-asset/
+description: "Learn more about large render blocking assets and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
```

Comment:
```suggestion
<Include name="early-adopter-note.mdx" />
```
Could you add this feature to the EA list here: https://docs.sentry.io/product/accounts/early-adopter/

Reply:
Good catch, done!